src/cpuEngine.cpp

```diff
@@ -21,6 +21,8 @@ CStack CpuEngine::_cstack;
 int CpuEngine::_signal;
 bool CpuEngine::_count_overrun;
 
+int CpuEngine::_target_cpu = -1;
```

Comment:
The option is specific to PerfEvents - I'd move it there.

Reply:
a21d908797d6025ce88177ac337ae9e872308d05